android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -70,11 +71,23 @@
         /**
          * Receive information about features found in a call to {@link #pickFeature(float, float)}
          * @param properties A mapping of string keys to string or number values
-         * @param positionX The horizontal screen coordinate of the center of the feature
-         * @param positionY The vertical screen coordinate of the center of the feature
+         * @param positionX The horizontal screen coordinate of the tapped location
+         * @param positionY The vertical screen coordinate of the tapped location
          */
         void onFeaturePick(Map<String, String> properties, float positionX, float positionY);
     }
+    /**
+     * Interface for a callback to receive information about labels picked from the map
+     */
+    public interface LabelPickListener {
+        /**
+         * Receive information about labels found in a call to {@link #pickLabels(float, float)}
+         * @param label The {@link TouchLabel} that has been selected
+         * @param positionX The horizontal screen coordinate of the tapped location
+         * @param positionY The vertical screen coordinate of the tapped location
+         */
+        void onLabelPick(TouchLabel label, float positionX, float positionY);
```

Comment:
Would `label.getCoordinate()` return the same longitude and latitude values for the icon label and the text label no matter which one is selected?

I'm just trying to avoid the case where the pin gets dropped in a different location depending on whether the icon or text is tapped.